smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaSink.java

```diff
@@ -79,6 +80,7 @@ public KafkaSink(Vertx vertx, KafkaConnectorOutgoingConfiguration config, KafkaC
 
         partition = config.getPartition();
         retries = config.getRetries();
+        deliveryTimeoutMs = Long.valueOf(kafkaConfigurationMap.get(ProducerConfig.DELIVERY_TIMEOUT_MS_CONFIG).toString());
```

Comment:
What if it's not set (and so `null`) ?

Reply:
Why is it a String BTW?

Reply:
done. The default value is retrieved from producer default config as an integer